doc/src/arch/reference.rst

```diff
@@ -28,10 +28,17 @@ The architecture tag contains the following tags:
 
 Recognized BLIF Models (<models>)
 ---------------------------------
-The ``<models>`` tag contains ``<model name="string">`` tags.
+The ``<models>`` tag contains ``<model name="string" never_prune="string">`` tags.
 Each ``<model>`` tag describes the BLIF ``.subckt`` model names that are accepted by the FPGA architecture.
 The name of the model must match the corresponding name of the BLIF model.
 
+The never_prune flag of the model is not strictly required and it can assume the following values:
```

Comment:
I think you could reword this comment to be a bit more clear.

"The never_prune flag is optional and can be either:"

... <text on options>

"Normally blocks with no output nets are pruned away by the netlist sweepers in vpr (removed from the netlist); this is the default behaviour.  If never_prune = true is set on a model, then blocks that are instances of that model will not be swept away during netlist cleanup. This can be helpful for some special blocks like I/Os <...add motivating case...> so space on the chip is still reserved for them, despite them not being connected to anything."

Reply:
Done, slightly changed also the suggested modification.